src/ICDF.h

```diff
@@ -0,0 +1,28 @@
+#pragma once
+
+
+#include "Constants.h"
+
+class ICDF
+{
+	double cdf_values_[CDF_RES + 1];
+
+public:
+
+	void SetDefaults(int startValue);
```

Comment:
Use `snake_case` for function names and argument variables.

`void set_defaults(int start_value);`

Reply:
Done